hera_mc/tests/test_correlator.py

```diff
@@ -1484,79 +1482,35 @@ def test_get_node_snap_from_serial_multiple_locs(mcsession):
     connection.start_gpstime = 1230375618
     mcsession.add(connection)
     mcsession.commit()
-    node, snap_loc_num = checkWarnings(
-        mcsession._get_node_snap_from_serial, ['SNPD000703'],
-        message='Multiple snap location numbers returned')
-    assert node == 701
-    assert snap_loc_num is None
+    pytest.raises(ValueError, mcsession._get_node_snap_from_serial, ['SNPD000703'])
 
 
-def test_get_node_snap_from_serial_multiple_nodes(mcsession):
-    """Test multiple node numbers."""
-    connection = cm_partconnect.Connections()
-    connection.upstream_part = 'SNPD000703'
-    connection.up_part_rev = 'A'
-    connection.downstream_part = 'N700'
-    connection.down_part_rev = 'A'
-    connection.upstream_output_port = 'rack'
-    connection.downstream_input_port = 'loc3'
-    connection.start_gpstime = 1230375618
-    mcsession.add(connection)
-    mcsession.commit()
-    node, snap_loc_num = checkWarnings(
-        mcsession._get_node_snap_from_serial, ['SNPD000703'],
-        message='Multiple node connections returned')
-    assert node is None
-    assert snap_loc_num is None
-
-
-def test_get_node_snap_from_serial_multiple_noderevs(mcsession):
-    """Test multiple node revisions."""
-    # add part for new revision
+def test_get_node_snap_from_serial_multiple_locs(mcsession):
```

Comment:
This test has the same name as the test before. Make sure the name isn't a duplicate and matches what is being tested.

Reply:
Renamed to incorporate info that multiple revs are checked.

Reply:
I'm not entirely sure what this one is about.  I did change the name to better describe the test.